frontend/benefit/handler/src/components/applicationReview/ApplicationReview.sc.ts

```diff
@@ -122,6 +144,12 @@ export const $Highlight = styled.div`
 `;
 
 export const $InfoNeededBar = styled.div`
+  position: relative;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+  z-index: 999
```

Comment:
Semicolon missing here, probably effectively disabling both this and the following CSS rule.